src/bundle/Microsoft.Kiota.Bundle.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Sdk Name="Microsoft.DotNet.PackageValidation" Version="1.0.0-preview.7.21379.12" />
+
+  <PropertyGroup>
+    <Description>Kiota Bundle package providing default implementations for client setup.</Description>
+    <AssemblyTitle>Kiota Bundle package for dotnet</AssemblyTitle>
+    <!-- NET 5 target to be removed on next major version-->
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0;net8.0</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\abstractions\Microsoft.Kiota.Abstractions.csproj" />
+    <ProjectReference Include="..\authentication\azure\Microsoft.Kiota.Authentication.Azure.csproj" />
```

Comment:
```suggestion
```

Reply:
Correct. No need for including the auth library.